api-reference/50 Common/Object Structures/ExcelPivotGridCell/area.md

```diff
@@ -4,7 +4,7 @@ type: String
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) to which a cell belongs.
```

Comment:
Since the ExcelPivotGridCell object describes one cell, all its fields describe this same cell. Hence, in all the field descriptions, the noun "cell" should have a definite article.